src/lwc/geFormFieldLookup/geFormFieldLookup.js

```diff
@@ -165,14 +189,30 @@ export default class GeFormFieldLookup extends LightningElement {
 
     @api
     setSelected(lookupResult) {
-        const lookup = this.template.querySelector('c-ge-autocomplete');
-        lookup.setLookUpData(lookupResult);
+       if (lookupResult.value === null) {
```

Comment:
I think this block should be one more space indented (through line 200)

Reply:
👍🏽